homeassistant/components/cover/mqtt.py

```diff
@@ -99,6 +142,18 @@ def async_added_to_hass(self):
         This method is a coroutine.
         """
         @callback
+        def tilt_updated(topic, payload, qos):
+            """The tilt was updated."""
+            if (payload.isnumeric() and
+                    self._tilt_min <= int(payload)
+                    and int(payload) <= self._tilt_max):
+                tilt_range = self._tilt_max - self._tilt_min
+                level = round(float(payload) / tilt_range * 100.0)
+                self._tilt_value = level
+                _LOGGER.info("Tilt value set to " + str(self._tilt_value))
```

Comment:
The log isn't necessary, since a log message will be emitted for the state change.